package.json

```diff
@@ -87,7 +87,9 @@
     "rimraf": "^5.0.1",
     "standard-version": "^9.5.0",
     "supertest": "^6.3.3",
-    "ts-jest": "^29.0.5",
+    "ts-jest": "^29.2.5",
+    "tsc-alias": "^1.8.10",
+    "tsconfig-paths": "^4.2.0",
```

Comment:
It seems that you are not using this package in the repo

Reply:
fixed